report/src/main/webapp/assets/sass/_buttons.scss

```diff
@@ -67,9 +67,7 @@
     line-height: 33px;
   }
 
-  i {
+  svg {
```

Comment:
Why were all the styles changed from "i" to "svg"? Seems like a lot of changes, is all styling ok throughout the app? Maybe this is something fontawesome specific that I don't know about :P